.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
I don't suppose you could remove your editor settings? Perhaps put .vscode in the .gitignore?